src/main.cpp

```diff
@@ -2446,16 +2446,10 @@ void static UpdateTip(CBlockIndex *pindexNew)
     }
 
     // Next, check every on every block for EB < 32MB and force this as the minimum because this is a consensus issue
-    // Although OP_RETURN size is not consensus, enforce the new minimum size on every block so that the expectation
-    // of relay for any tx < 220 bytes is met by this node.
     if (IsMay152018Enabled(chainParams.GetConsensus(), pindexNew))
     {
         if (miningForkEB.value > excessiveBlockSize)
             excessiveBlockSize = miningForkEB.value;
-        // Bump OP_RETURN size:
-        if (nMaxDatacarrierBytes < MAX_OP_RETURN_MAY2018)
-            nMaxDatacarrierBytes = MAX_OP_RETURN_MAY2018;
-        settingsToUserAgentString();
```

Comment:
we still need settingsToUserAgentString(); if excessiveBlockSize changed... how about only running it if EB changed:

if (miningForkEB.value > excessiveBlockSize)
{
      excessiveBlockSize = miningForkEB.value;
      settingsToUserAgentString();
}

Reply:
you are right indeed. fixing.

Reply:
fixed